src/screens/SearchScreen.js

```diff
@@ -0,0 +1,175 @@
+        let preparedName = _.capitalize(buttonName);
+
+        if (Platform.OS === 'ios') {
+            return (
+                <Button
+                    title={preparedName}
+                    onPress={toggleButton}
+                    backgroundColor="rgba(0,0,0,0)"
+                    color="rgba(0,122,255,1)"
+                />
+            );
+        } else if (Platform.OS === 'android') {
+            // android-specific code for navigation here
+        }
+    };
+
+    static renderRightNavButton = (navigate) => {
+        if (Platform.OS === 'ios') {
+            return (
+                <Button
+                    title="Filter"
+                    onPress={() => navigate('filter')}
+                    backgroundColor="rgba(0,0,0,0)"
+                    color="rgba(0,122,255,1)"
+                />
+            );
+        } else if (Platform.OS === 'android') {
+            // android-specific code for navigation here
+        }
+    };
+
+    static navigationOptions = (props) => {
+        const {navigation: {navigate, state: {params = {}}}} = props;
+
+        return {
+            title: 'Search',
+            headerTitle: 'Find a Site',
+            headerLeft: SearchScreen.renderLeftNavButton(params),
+            headerRight: SearchScreen.renderRightNavButton(navigate),
+            tabBarIcon: ({focused, tintColor}) => (
+                <Icon type='ionicon' name={focused ? 'ios-search' : 'ios-search-outline'} size={25} color={tintColor}/>)
+        }
+
+    };
+
+    regionDeltaIsAcceptable = (newRegion) => {
+        const longDeltaIsAcceptable = (Math.abs(newRegion.longitudeDelta - this.state.region.longitudeDelta) < 10);
+        const latDeltaIsAcceptable = (Math.abs(newRegion.latitudeDelta - this.state.region.latitudeDelta) < 10);
+
+        return longDeltaIsAcceptable && latDeltaIsAcceptable;
+    };
+
+    onRegionChangeComplete = (newRegion) => {
+
+        if (!this.regionDeltaIsAcceptable(newRegion)) {
+            return;
+        }
+        this.setState({
+            region: newRegion
+        })
+    };
+
+    renderMap = () => {
+        const {fillScreen, spinnerContainerStyle} = styles;
+
+        if (this.state.mapLoaded) {
+            return (
+                <MapView
+                    style={fillScreen}
+                    region={this.state.region}
+                    onRegionChangeComplete={this.onRegionChangeComplete}
+                />
+            )
+        } else {
+            return (
+                <View style={[fillScreen, spinnerContainerStyle]}>
+                    <ActivityIndicator size="large"/>
+                </View>
+            );
+        }
+    };
+
+    renderList = () => {
+        return (
+            <Card>
+                <CardSection>
+                    <Text>
+                        List Results of Sites
+                    </Text>
+                </CardSection>
+            </Card>
+        );
+    };
```

Comment:
Right now `renderMap` and `renderList` are hard-coded in. After I finish the refactor, they'll appropriately call the `SearchMap` and `SearchList` components that you saw above.